tests/toolkit/test_developer.py

```diff
@@ -1,4 +1,18 @@
 from pathlib import Path
+
+def test_system_without_hints(temp_dir, developer_toolkit):
+    (temp_dir / 'prompts').mkdir(parents=True, exist_ok=True)
```

Comment:
nit: i don't believe this is necessary (the prompts dir system is loaded from is part of the source code, not the cwd)